packages/table/src/common/_variables.scss

```diff
@@ -1,6 +1,7 @@
 // Copyright 2016 Palantir Technologies, Inc. All rights reserved.
 
 @import "~@blueprintjs/core/src/common/variables";
```

Comment:
I guess here is not going to have any effect... but customs are better at the begining so the rules with `!defaults` can be redefined.

```suggestion
@import "./customs";
```

Reply:
But, if I import just './customs', I am not importing the variables from core that I wanted

Reply:
So, where do you propose to import the `@import "~@blueprintjs/core/src/common/variables";` ?

Reply:
@crispamares for instance, in `select`, we do first `@import "~@blueprintjs/core/src/common/variables";` in `_select.scss` file

Reply:
Yes we need the two... It is a shame that here I can't suggest multiline changes... that's why in the next suggestion I add the variables again ^_^